diff --git a/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol b/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol
index 2c8aa437..8613b9b3 100644
--- a/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol
+++ b/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol
@@ -28,12 +28,6 @@ import "../../../proxy/utils/Initializable.sol";
  */
 abstract contract ERC20VotesUpgradeable is Initializable, IVotesUpgradeable, ERC20PermitUpgradeable {
     using ERC20VotesStorage for ERC20VotesStorage.Layout;
-    function __ERC20Votes_init() internal onlyInitializing {
-        __Context_init_unchained();
-        __EIP712_init_unchained(name, "1");
-        __ERC20Votes_init_unchained();
-    }
-
     function __ERC20Votes_init_unchained() internal onlyInitializing {
     }
     struct Checkpoint {
