diff --git a/contracts/token/common/ERC2981Upgradeable.sol b/contracts/token/common/ERC2981Upgradeable.sol
index f62ce8dd..27a79add 100644
--- a/contracts/token/common/ERC2981Upgradeable.sol
+++ b/contracts/token/common/ERC2981Upgradeable.sol
@@ -45,7 +45,7 @@ abstract contract ERC2981Upgradeable is Initializable, IERC2981Upgradeable, ERC1
     }
 
     /**
-     * @inheritdoc IERC29810
+     * @inheritdoc IERC2981Upgradeable
      */
     function royaltyInfo(uint256 _tokenId, uint256 _salePrice)
         external
